store/tikv/gcworker/gc_worker_test.go

```diff
@@ -187,6 +188,29 @@ func (s *testGCWorkerSuite) TestPrepareGC(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(ok, IsTrue)
 
+	// Check gc life time small than min.
+	s.oracle.AddOffset(time.Minute * 40)
+	err = s.gcWorker.saveDuration(gcLifeTimeKey, time.Minute)
+	c.Assert(err, IsNil)
+	ok, _, err = s.gcWorker.prepare()
+	c.Assert(err, IsNil)
+	c.Assert(ok, IsTrue)
+	lifeTime, err := s.gcWorker.loadDuration(gcLifeTimeKey)
+	c.Assert(err, IsNil)
+	c.Assert(*lifeTime, Equals, gcMinLifeTime)
+
+	// Check gc life time small than config.max-txn-use-time
+	s.oracle.AddOffset(time.Minute * 40)
+	config.GetGlobalConfig().TiKVClient.MaxTxnTimeUse = 20*60 - 10 // 20min - 10s
+	err = s.gcWorker.saveDuration(gcLifeTimeKey, time.Minute)
+	c.Assert(err, IsNil)
+	ok, _, err = s.gcWorker.prepare()
+	c.Assert(err, IsNil)
+	c.Assert(ok, IsTrue)
+	lifeTime, err = s.gcWorker.loadDuration(gcLifeTimeKey)
+	c.Assert(err, IsNil)
+	c.Assert(*lifeTime, Equals, 20*time.Minute)
+
```

Comment:
Please also check if the configured lifetime is greater than either 10m or MaxTxnTimeUse+10s, it will be left unchanged after preparing.

Reply:
done.